### DISCUSSION TITLE
What's wrong with the Symmetric Rank one method?

### DISCUSSION THREAD

**Discussion by mcgarnett** [coin-or/Ipopt#561] [General]
The manual mentions that it's "not working well" but it's working great for my problem, showing much more robust convergence than L-BFGS. Any caveats about using it?

**Comment by svigerske** [comment]
I guess that it didn't work well in general. There are a number of TODOs in the code that indicate that the implementation should be improved: https://github.com/coin-or/Ipopt/blob/stable/3.14/src/Algorithm/IpLimMemQuasiNewtonUpdater.cpp#L523
